Makefile

```
@@ -0,0 +1,62 @@
+.PHONY: all
+all: build
+
+.PHONY: distclean
+distclean: clean
+
+clean: clean-deb
+	find ./ -type f -name "*.onnx" -exec rm {} \;
+	find ./ -type f -name "*.rknn" -exec rm {} \;
+
+.PHONY: clean-deb
+clean-deb:
+	rm -rf debian/.debhelper debian/rknn-model-zoo-rk3566 debian/rknn-model-zoo-rk3568 debian/rknn-model-zoo-rk3588 debian/debhelper-build-stamp debian/files debian/*.debhelper.log debian/*.postrm.debhelper debian/*.substvars debian/*.debhelper.log
+
+#
+# Test
+#
+.PHONY: test
+test:
+
+#
+# Build
+#
+yolov8_rknn: yolov8_rk3588.rknn yolov8_rk3568.rknn yolov8_rk3566.rknn
+
+yolov8n.onnx:
```

```suggestion
examples/yolov8/model/yolov8n.onnx:
```
makefile的对象是完整路径，否则你再次运行会再次下载&转换